src/libraries/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -763,14 +763,16 @@ public void SetThreadPrincipal()
 
         private void CopyTestAssemblies()
         {
-            string destTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "AssemblyResolveTestApp", "AssemblyResolveTestApp.dll");
+            string curDirectory = PlatformDetection.IsNotAppleMobile ? Environment.CurrentDirectory : Environment.GetFolderPath(Environment.SpecialFolder.InternetCache); 
```

Comment:
I'd only do the alternate path for `tvOS` where we know that `Environment.CurrentDirectory` isn't writable, and use `Path.GetTempPath()` there.

Reply:
Actually, I wonder how this even works given that the only non-disabled test on mobile, `ExecuteAssembly()`, uses `Environment.CurrentDirectory` to create the path to `TestAppOutsideOfTPA.exe`